src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 8, max = 32)
     private String password;
+    @Size(min = 8, max = 32)
+    private String repeatPassword;
```

Comment:
```suggestion
    private String password;
    private String repeatPassword;
```
If you need any additional check for password add it to PasswordValidator. Let's have one place for all checks
![image](https://github.com/mate-academy/jv-spring-security/assets/20326858/913f0907-3f7b-4b34-b64b-e8b3ba174235)